stores/message.js

```diff
@@ -274,6 +306,21 @@ export const useMessageStore = defineStore({
     async intend(id, outcome) {
       await api(this.config).message.intend(id, outcome)
     },
+    async fetchActivePostCount() {
+      const authStore = useAuthStore()
+      const userUid = authStore.user?.id
+
+      return api(this.config).message.fetchByUser(userUid, false)
```

Comment:
The active post count includes only active messages.  So when you are fetching the messages for the count, you should pass `true`.